packages/iptables.rb

```diff
@@ -9,15 +9,15 @@ class Iptables < Package
 
   binary_url ({
      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/iptables-1.6.1-1-chromeos-x86_64.tar.xz',
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/iptables-1.6.1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/iptables-1.6.1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/iptables-1.6.1-chromeos-i686.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/iptables-1.6.1-1-chromeos-armv7l.tar.xz',
```

Comment:
Iptables didn't need new binaries. The old ones work just fine. The only thing changed was the x86_64 one because of not using `/usr/local/lib64`.